promptsource/templates/xsum/templates.yaml

```diff
@@ -10,15 +10,16 @@ templates:
       ===
 
 
-      Given the above document, generate a summary: ||| {{summary}}'
+      Write a summary of the text above : ||| {{summary}}'
```

Comment:
I guess the author of this particular template wanted to compare between prompts with or without "one sentence." 
If that's the case, this one line change shall be rejected.